### PR TITLE
Update _strings_en.yml

### PR DESCRIPTION
**Pull request by QuickBASIC**
I feel like the language used on the main site is either wordy, confusing, or is written in a more casual writing style. We need to make sure that the information that's eventually translated is good technical writing (simple and easy to understand). It can be off-putting for new users for the language on the site to be less professional.

### CODE REVIEW

File: _strings_en.yml [hunk 1]
```diff
@@ -65,9 +65,9 @@ index:
...
-  private_text: "Monero uses a cryptographically sound system that allows you to send and receive funds without your @transactions being publicly visible on the @blockchain (the distributed ledger of transactions). This ensures that your purchases, receipts, and other transfers remain private by default."
-  untraceable_text: "By taking advantage of @ring-signatures, a special property of certain types of cryptography, Monero enables untraceable transactions. This means it's ambiguous which funds have been spent, and thus extremely unlikely that a transaction could be linked to particular user."
-  secure_text: "Using the power of a distributed peer-to-peer @consensus network, every transaction is cryptographically secured. Individual @accounts have a 25 word @mnemonic-seed displayed when created, which can be written down to back up the @account. Account files are encrypted with a passphrase to ensure they are worthless if stolen."
+  private_text: "Monero uses hidden amounts, origins and destinations of @transactions. This means that nobody knows how much you're sending in a transaction except you. This ensures that your purchases and other transfers remain private by default."
```
Comment (mattcode55) [inline _strings_en.yml]: >your transcation remains private and untraceable to you.

Transaction is misspelt, and I think that the "to you" at the end should be removed, or replaced with "and cannot be traced back to you"

Comment (QuickBASIC) [inline _strings_en.yml]: Fixed.

**Comment by QuickBASIC** [comment]
@luigi1111 #42 

> I realize (realise?) a lot of this is is personal style, but some of the sentence structure was rather awkward.
> I'm not in love with the "untraceable_text" yet; suggestions welcome.

Two years later and someone finally took a crack at it. I would appreciate your opinion on the wording.

**Comment by tyrionmcmaster** [comment]
Good.

**Pull request merged**